README.md

```diff
@@ -2,55 +2,52 @@
 
 [![CircleCI](https://circleci.com/gh/codacy/helm-poll/tree/master.svg?style=svg)](https://circleci.com/gh/codacy/helm-poll/tree/master)
 
-An helm plugin to poll for a release status.
+A Helm plugin to poll for a release status.
 
 ## Project Goals
 
-As per the [Helm documentation](https://helm.sh/docs/helm/helm_status/), a release can have the following final statuses:
+As per the [Helm documentation](https://helm.sh/docs/helm/helm_status/), a release can have the following final states:
```

Comment:
The Helm documentation says that the status of a named release includes:

- last deployment time
- k8s namespace in which the release lives
- state of the release

So these five different results are actually the **state** of the release. :)